.github/workflows/coverage.yml

```diff
@@ -31,8 +31,12 @@ jobs:
         run: |
           cd "${CARGO_HOME}/bin"
           curl -sL https://github.com/xd009642/tarpaulin/releases/download/${CARGO_TARPAULIN_VERSION}/cargo-tarpaulin-${CARGO_TARPAULIN_VERSION}-travis.tar.gz | tar xzvf -
-      - run: cargo tarpaulin --locked --workspace --exclude=linkerd2-proxy --exclude=linkerd-app-integration --no-run
-      - run: cargo tarpaulin --locked --workspace --exclude=linkerd2-proxy --exclude=linkerd-app-integration --skip-clean --ignore-tests --no-fail-fast --out=Xml
-      - run: cargo tarpaulin --locked --workspace --package=linkerd-app-integration --no-default-features --skip-clean --no-run
-      - run: cargo tarpaulin --locked --workspace --package=linkerd-app-integration --no-default-features --skip-clean --ignore-tests --no-fail-fast --out=Xml
+      # XXX(ver) AFAICT, Tarpaulin doesn't allow us to compose a report over multiple invocations,
+      # so we have to choose between getting coverage from unit tests and integration tests (since
+      # integration tests require --no-default-features to avoid flakiness). Currently the
+      # integration tests seem to cover more code, so we skip the unit tests for now :(.
+      #- run: cargo tarpaulin --locked --workspace --exclude=linkerd2-proxy --exclude=linkerd-app-integration --no-run
+      #- run: cargo tarpaulin --locked --workspace --exclude=linkerd2-proxy --exclude=linkerd-app-integration --skip-clean --ignore-tests --no-fail-fast --out=Xml
```

Comment:
hmm, that's kind of a bummer. Looking at the docs it looks like we might be able to define different configurations (with different featuresets) in [Tarpaulin's config file](https://github.com/xd009642/tarpaulin#config-file), I wonder if we could use that instead of separate runs?

Reply:
Cool! If you can help figure that out, would be happy to use it :)